examples/fingerprint_picturetest.py

```diff
@@ -0,0 +1,254 @@
+# SPDX-FileCopyrightText: 2021 ladyada for Adafruit Industries
+# SPDX-License-Identifier: MIT
+
+# Added 'View Print' and 'Preview and Find Print' functions to
+# example code fingerprint_simpletest.py
```

Comment:
We typically use triple quotes for docstrings, but I don't think this is a gating change.  More importantly, I would specify that this example only works on single board computers with the use of Blinka, our CircuitPython compatibility layer.